test/cluster/blue-green-deployment/setup.td

```diff
@@ -27,14 +27,18 @@ ALTER SYSTEM SET enable_unmanaged_cluster_replicas = true;
 > DROP SCHEMA IF EXISTS prod CASCADE
 > DROP SCHEMA IF EXISTS prod_deploy CASCADE
 
-> CREATE SCHEMA prod
-> CREATE SOURCE prod.counter IN CLUSTER prod FROM LOAD GENERATOR counter (TICK INTERVAL '0.01s')
-> CREATE MATERIALIZED VIEW prod.counter_mv IN CLUSTER prod AS SELECT count(*) FROM prod.counter
-> CREATE DEFAULT INDEX IN CLUSTER prod ON prod.counter
-> CREATE SOURCE prod.tpch
-  IN CLUSTER prod
+# For now sources are not considered in blue-green deployments, so create them separately
+> DROP SOURCE IF EXISTS counter
+> CREATE SOURCE counter FROM LOAD GENERATOR counter (TICK INTERVAL '0.01s')
+
+> DROP SOURCE IF EXISTS tpch
+> CREATE SOURCE tpch
   FROM LOAD GENERATOR TPCH (SCALE FACTOR 1)
```

Comment:
As discussed in Slack, we'll probably want to reduce the scale factor to avoid the test from out because the dataflows take too long to hydrate.